s3tests_boto3/functional/test_s3.py

```diff
@@ -8838,83 +8838,44 @@ def trigger(self):
             self.result = self.action()
 
 
-@pytest.mark.skip(reason="Potential Bug")
-def test_multipart_resend_first_finishes_last():
+@pytest.mark.skip(reason="https://github.com/nspcc-dev/neofs-s3-gw/issues/901")
+def test_multipart_resend():
```

Comment:
This change (whole commit) deserves better explanation in the commit message.

Reply:
added additional description to the commit